SpringProject/src/main/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+spring:
+  application:
+    name: SpringProject
+  datasource:
+    platform: mysql
+    driverClassName: com.mysql.cj.jdbc.Driver
+    url: jdbc:mysql://localhost:3306/conference_finder?autoreconnect=true
+    password: password
+    username: root
+
+#flyway:
```

Comment:
The same goes for this: If you're not using it, it's better to not comment it out but just to remove it ;)